### PR TITLE
Add support for DD_APPEND_SLASH

### PR DESCRIPTION
**Pull request by kiblik**
Avoid situations like https://owasp.slack.com/archives/C2P5BA8MN/p1705702146187379

Better context description: https://learndjango.com/tutorials/trailing-url-slashes-django

Documentation: https://docs.djangoproject.com/en/4.2/ref/settings/#append-slash

### CODE REVIEW

File: dojo/settings/settings.dist.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
     DD_SECURE_CONTENT_TYPE_NOSNIFF=(bool, True),
     DD_CSRF_COOKIE_SAMESITE=(str, 'Lax'),
     DD_SESSION_COOKIE_SAMESITE=(str, 'Lax'),
+    DD_APPEND_SLASH=(bool, False),
```
Comment (Maffooch) [inline dojo/settings/settings.dist.py:43]: We should default to True here to match what it has always been for folks as well as the default set by Django

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/settings/settings.dist.py (hunk 1 above)
Comment (kiblik) [inline dojo/settings/settings.dist.py:43]:
```suggestion
    DD_APPEND_SLASH=(bool, True),
```

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Comment by kiblik** [comment]
Hi @grendel513 / @devGregA / @cneill / @blakeaowens. I would like to ask for a review.

**Review by grendel513 — APPROVED** [review]

**Review by blakeaowens — APPROVED** [review]

**Pull request merged**